src/game.c

```diff
@@ -933,7 +934,19 @@ bool game_load_save(const utf8 *path)
 		return false;
 	}
 
-	bool result = game_load_sv6(rw);
+	char *extension = strrchr(path, '.');
```

Comment:
That applies to the command line parameter parsing code as well then. (`game.c`, line 937)
I'll use `get_file_extension_type()` and I'll also adjust the closing of `rw`.